CMakeLists.txt

```diff
@@ -436,7 +436,24 @@ ENDIF (ARCH MATCHES "arm*" AND (NOT QT_ANDROID) )
 
 # Search for opengles, short of running a program to test the speed
 # of acceleration, I simply use gles on "native linux" arm systems
-IF (ARCH MATCHES "arm*" AND (NOT QT_ANDROID) AND CMAKE_SYSTEM_NAME MATCHES ".*Linux" )
+IF(DEFINED _wx_selected_config)
+  MESSAGE (STATUS "selected config ${_wx_selected_config}")
+  IF(_wx_selected_config MATCHES "androideabi-qt")
+    MESSAGE (STATUS "Building for wxQt-Android")
+    MESSAGE (STATUS "Qt_Base: " ${Qt_Base})
+    MESSAGE (STATUS "wxQt_Base/Build: " ${wxQt_Base} "/" ${wxQt_Build})
+    SET(QT_ANDROID "ON")
+  ENDIF(_wx_selected_config MATCHES "androideabi-qt")
+ENDIF(DEFINED _wx_selected_config)
+
+
+IF((_wx_selected_config MATCHES "qt-armv7"))
+  SET(wxWidgets_USE_LIBS base core xml html adv aui)
+ELSE()
+  SET(wxWidgets_USE_LIBS net xml html adv aui core base webview)
+ENDIF()
```

Comment:
Do we still need to do the above if we later try to recheck without webview in case it is missing (L277-L585)?

Reply:
Not really, I guess...  Branch is rewritten, focused on a minimal beta freeze change.